src/clr-angular/utils/loading/loading.module.ts

```diff
@@ -6,13 +6,15 @@
 import {CommonModule} from "@angular/common";
 import {NgModule, Type} from "@angular/core";
 
-import {ClrLoading} from "./loading";
+import {ClrLoading, ClrLoadingState} from "./loading";
 
 export const CLR_LOADING_DIRECTIVES: Type<any>[] = [ClrLoading];
 
 @NgModule({imports: [CommonModule], declarations: [CLR_LOADING_DIRECTIVES], exports: [CLR_LOADING_DIRECTIVES]})
 export class ClrLoadingModule {}
 
+export const loadingState = ClrLoadingState;
```

Comment:
Not sure what this renamed export is for. I might have missed it somewhere. But it looks like it can be safely removed.